docs/WalletGuide.md

```diff
@@ -64,7 +64,7 @@ let anchor = wallet.anchor({ homeDomain: "testanchor.stellar.org" });
 Get anchor information from a TOML file.
```

Comment:
I think we should remove this md file in favor of stellar-docs (maybe part of WAL-972)